packages/legend-query/src/stores/QueryBuilderState.ts

```diff
@@ -123,10 +131,12 @@ export class QueryBuilderState {
   constructor(
     applicationStore: ApplicationStore<QueryConfig>,
     graphManagerState: GraphManagerState,
+    queryBuilderConfig?: QueryBuilderConfig | undefined,
```

Comment:
As discussed, let's change this to required and not expose `setConfig`, could we?